eagle-core/eagle-alert-parent/eagle-alert/alert-common/src/main/java/org/apache/eagle/alert/engine/model/AlertStreamEvent.java

```diff
@@ -39,6 +39,7 @@
     private long createdTime;
     private String category;
     private AlertSeverity severity = AlertSeverity.WARNING;
+    private boolean duplicationChecked = false;
```

Comment:
I think it is impossible to change it, because there is no other better way to [design](https://docs.oracle.com/javase/specs/jls/se7/html/jls-4.html#jls-4.12.5) it and redesignning it will bring too much impact ...